src/main.rs

```diff
@@ -36,23 +41,28 @@ fn publish(docs_path: &str,
            sign: bool,
            push_remote: &str,
            push_branch: &str,
+           shell: &mut MultiShell,
            dry_run: bool)
            -> Result<bool, FatalError> {
-    println!("Publishing Sphinx docs to GitHub Pages.");
+    try!(shell.say("Publishing Sphinx docs to GitHub Pages.", color::BLUE));
```

Comment:
To me this falls into the "verbose" category. There's a [verbose fn](https://github.com/rust-lang/cargo/blob/5716f32d78b1baf629e055573a682a6730936b4b/src/cargo/core/shell.rs#L90) on shell, but I don't know how to call this while proxying the Result.

```
try!(shell.verbose(|| shell.say(...)));
```

doesn't seem to do the trick.